### PR TITLE
fix server test.

### PR DESCRIPTION
**Pull request by huachaohuang**
Seems there are some problems with our updated etcd, try to fix CI first and investigate it later.

### CODE REVIEW

**Comment by huachaohuang** [comment]
@overvenus @qiuyesuifeng @siddontang

**Comment by overvenus** [comment]
LGTM

There is a similar patch in #302 , we should fix it too.
https://github.com/pingcap/pd/pull/302/commits/8deea1aaf427c33dab82724de824ea87c6b97014#diff-d6fee93ad3d6a100bd53f8a0a153c865R70

**Comment by c4pt0r** [comment]
LGTM

File: server/server_test.go [hunk 1]
```diff
@@ -183,16 +183,28 @@ func (s *testLeaderServerSuite) TestLeader(c *C) {
 	svr.Close()
 	delete(s.svrs, leader1.GetAddr())
 
+	// Create a client without the leader1's endpoints.
+	endpoints := make([]string, 0, 2)
+	for _, svr := range s.svrs {
+		endpoints = append(endpoints, svr.GetEndpoints()...)
+	}
+	client, err := clientv3.New(clientv3.Config{
```
Comment (qiuyesuifeng) [inline server/server_test.go]: Can we remove s.client?

**Pull request merged**